scipy/sparse/tests/test_base.py

```diff
@@ -807,7 +853,7 @@ def test_trace(self):
             assert_equal(A.trace(offset=k), B.trace(offset=k))
 
     def test_reshape(self):
-        # This first example is taken from the lil_matrix reshaping test.
+        # This first example is taken from the lil_object reshaping test.
```

Comment:
This test was moved to TestCommon from TestLIL in [#7945](https://github.com/scipy/scipy/pull/7945/files). But it was also changed in that process. There doesn't seem to be anything this comment points to that would be helpful. So I have removed the comment. That PR #7945 is the place to look for the history of this test (the code hasn't been changed since it was merged).